server/src/main/java/io/druid/server/log/FilteredRequestLoggerProvider.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.server.log;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import io.druid.server.RequestLogLine;
+
+import javax.validation.constraints.NotNull;
+import java.io.IOException;
+
+/**
+ */
+@JsonTypeName("filtered")
+public class FilteredRequestLoggerProvider implements RequestLoggerProvider
+{
+  @JsonProperty
+  @NotNull
+  private RequestLoggerProvider delegate = null;
+
+  @JsonProperty
+  private long queryTimeThreshold = 0;
```

Comment:
IMO it would be nice to be more verbose about units, something like `queryTimeThresholdMs` (if it is actually milliseconds in this case). And/or specify the units generously in class, constructor Javadocs, comments in configs, Druid docs.

Obviously `"query/time"` as well as a lot of other metric names, configuration parameters, variables, and fields in the Druid codebase also have this problem, but it is something that should be addressed separately (and I plan to do this). At least we can stop the old practice in the new code.